### PR TITLE
Add first pass at README

### PR DESCRIPTION
**Pull request by danhodos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+An [Alamofire](https://github.com/Alamofire/Alamofire) extension that automatically converts JSON into Swift objects using [Genome](https://github.com/LoganWright/Genome).
+
+## Installation
+
+### Cocoapods
+
+Add the following to your `Podfile`:
+
+```
+pod 'AlamofireGenome', '~> 1.0'
```
Comment (jdzak) [inline README.md]: What do you think of leaving the version off? I prefer locking to a specific version when there's a reason, but leaving it open if not.

**Comment by jdzak** [comment]
Also, for the local interation-y spec, having it as either an integration spec or sample project works for me. I think later down the road we can polish it and get rid of the hit to the network, but I think it'd be helpful to have somewhere in the meantime in case we want to verify the README code.

File: README.md (hunk 1 above)
Comment (danhodos) [inline README.md]: I dig that!

**Comment by danhodos** [comment]
> Also, for the local interation-y spec, having it as either an integration spec or sample project works for me.

A sample project is a pretty smart idea; do you mind if I do that as a separate PR, either way?

**Comment by jdzak** [comment]
Works for me.

**Pull request merged**